### PR TITLE
Context run/pressure test (74%!)

### PR DESCRIPTION
**Pull request by josephbirkner**
Added  `test_context_integrated.py`, `state.wait()`, `context.run_once()`:
* 74% Coverage for all ravestate Modules
* 88% for ravestate core module

Enables easy tests for specific state interactions in a context!

### CODE REVIEW

File: modules/ravestate/state.py [hunk 1]
```diff
@@ -132,6 +135,19 @@ def signal(self) -> Optional[Signal]:
             self._signal = s(f"{self.module_name}:{self.signal_name}")
         return self._signal
 
+    def wait(self, timeout=5.):
+        """
+        For for the state's activation function to be run at least once.
```
Comment (Toseban) [inline modules/ravestate/state.py]:
```suggestion
        Wait for the state's activation function to be run at least once.
```

File: test/modules/ravestate/test_context_integrated.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    ctx._run_once()
+    assert signal_b.wait()
+
+    # make sure that pressuring_state is pressuring specific_state
+    acts = ctx._state_activations(st=specific_state)
+    assert any(act.is_pressured() for act in acts)
+
+    ctx._run_once()
+    assert specific_state.wait()
```
Comment (Toseban) [inline test/modules/ravestate/test_context_integrated.py]: Awesome way of testing context runs :fire:

File: modules/ravestate/testfixtures.py [hunk 3]
```diff
@@ -23,7 +23,7 @@
 def state_fixture(mocker):
     @state(write=(DEFAULT_PROPERTY_ID,), read=(DEFAULT_PROPERTY_ID,))
     def state_mock_fn(ctx):
-        pass
+        ctx[DEFAULT_PROPERTY_ID] = "test"
```
Comment (Toseban) [inline modules/ravestate/testfixtures.py]: Is this used anywhere yet?
Maybe we can replace the hardcoded value with a variable

Comment (josephbirkner) [reply modules/ravestate/testfixtures.py]: Not sure about a lot of things in testfixtures.py, let's keep it on our radar.

**Pull request merged**